test/integration/targets/any_errors_fatal/tasks_after_always_block.yml

```
@@ -0,0 +1,27 @@
+- hosts: testhost,testhost2
+  gather_facts: no
+  tasks:
+  - block:
+    - fail:
+      when: inventory_hostname == 'testhost'
+    - debug: msg="still executing in block"
+    rescue:
+    - meta: clear_host_errors
+    always:
+    - debug: msg="executing in always"
+    any_errors_fatal: yes
+
+  - debug: msg="continue executing after recovering from failure"
+
+- hosts: testhost,testhost2
+  gather_facts: no
+  tasks:
+  - block:
+    - fail:
```

Tasks that are expected to fail should have a name starting with `EXPECTED FAILURE` to prevent the task from triggering unstable test results on Shippable.

```suggestion
    - name: EXPECTED FAILURE fail on testhost
       fail:
```